app/screens/integration_selector/index.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+
+import {observeCurrentTeamId} from '@queries/servers/system';
+
+import IntegrationSelector from './integration_selector';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+// See LICENSE.txt for license information.
```

Comment:
This seems out of place. Remove it.

Reply:
I think this was added by my ESLint, no sure why, I'll remove it.